docs/whatisk8s.md

```diff
@@ -134,11 +135,14 @@ This [design](design/principles.md) has enabled a number of other systems to bui
 
 #### Kubernetes is not:
 
-Kubernetes is not a traditional PaaS (Platform as a Service) system.
-* Kubernetes does not limit the types of applications supported. It does not dictate application frameworks, restrict the set of supported language runtimes, cater to only [12-factor applications](http://12factor.net/), nor distinguish "apps" from "services". Kubernetes aims to support an extremely diverse variety of workloads: if an application can run in a container, it should run great on Kubernetes.
-* Kubernetes does not provide middleware (e.g., message buses), databases (e.g., mysql), nor cluster storage systems (e.g., Ceph) as services.
-* Kubernetes does not have a click-to-deploy service catalog.
+Kubernetes is not a traditional, all-inclusive PaaS (Platform as a Service) system. We preserve user choice where it is important.
+* Kubernetes does not limit the types of applications supported. It does not dictate application frameworks (e.g., [Wildfly](http://wildfly.org/)), restrict the set of supported language runtimes (e.g., Java, Python, Ruby), cater to only [12-factor applications](http://12factor.net/), nor distinguish "apps" from "services". Kubernetes aims to support an extremely diverse variety of workloads, including stateless, stateful, and data-processing workloads. If an application can run in a container, it should run great on Kubernetes.
+* Kubernetes does not provide middleware (e.g., message buses), data-processing frameworks (e.g., Spark), databases (e.g., mysql), nor cluster storage systems (e.g., Ceph) as built-in services. Such applications run on Kubernetes.
+* Kubernetes does not have a click-to-deploy service marketplace.
 * Kubernetes is unopinionated in the source-to-image space. It does not deploy source code and does not build your application. Continuous Integration (CI) workflow is an area where different users and projects have their own requirements and preferences, so we support layering CI workflows on Kubernetes but don't dictate how it should work.
+* Kubernetes allows users to choose the logging, monitoring, and alerting systems of their choice. (Though we do provide some integrations as proof of concept.)
+* Kubernetes does not provide nor mandate a comprehensive application configuration language/system (e.g., [jsonnet](https://github.com/google/jsonnet)).
+* Kubernetes does not provide nor adopt any comprehensive machine configuration, maintenance, and management system (e.g., [Chef](https://www.chef.io/chef/), [Puppet](https://puppetlabs.com/solutions/configuration-management), [Ansible](http://www.ansible.com/), [Salt](http://saltstack.com/)).
```

Comment:
I'm actually not sure that Chef/Puppet/Ansible are the right comparisons here (or at least its not exhaustive) we also don't provide machine health monitoring, maintenance or eviction of sick machines.